### DISCUSSION TITLE
Restore elevation when the internet briefly disconnects and perhaps in other cases

### DISCUSSION THREAD

**Discussion by sn-o-w** [rustdesk/rustdesk#5867] [Feature Request]
Follow up of https://github.com/rustdesk/rustdesk/issues/5864

If a user connects to another computer, then requests Elevation, and if the internet briefly disconnects and they connect back via RustDesk, RuskDesk may be unable to work properly, being stuck on the local machine, and being unable to operate the mouse and keyboard on the remote side. 

Looks like the only solution for this would be to request Elevation again. A regular user would remain stuck in this scenario if they don't know they need to request Elevation again. I consider it as not ideal by default to let users remain confused.

I would appreciate at least an option in Settings, such as "Persist Elevation when reconnecting" or something like that. 🤔

**Comment by sn-o-w** [comment]
Also, by the way, you don't have to struggle with anything like this if you use TeamViewer. TeamViewer will simply start to work again after the internet connects back.

If you want to be competitive against TeamViewer, maybe you should at least give us an option to not request Elevation every time something happens out of the user's control, like the Internet briefly disconnecting. 🤔

Just my two cents.

**Comment by rustdesk** [comment]
We have our priority list, won't change because of  someone's opinion, because every feature request will say as you. But we do put it with high priority if we receive a lot of feedback.

**Reply by sn-o-w** [reply]
Huh? I never said you needed to change your priority list based on my or someone's opinion, lol. 🤨 I just expressed my own opinion; you told me to submit a feature request. 😶 It's more productive to talk ideas and opinions instead of "feature request: persist elevation" and nothing more than that...
